src/Beman/Example/CMakeLists.txt

```
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(TARGET_LIBRARY ${CMAKE_PROJECT_NAME})
+set(TARGET_LIBRARY_ALIAS "Beman::Example")
+set(TARGET_LIBRARY_BINARY "lib${TARGET_LIBRARY}")
+
+add_library(${TARGET_LIBRARY} STATIC)
+add_library(${TARGET_LIBRARY_ALIAS} ALIAS ${TARGET_LIBRARY})
```

```suggestion
add_library(beman.example STATIC)
add_library(beman::example ALIAS beman.example)
```

IMO it is much simpler to avoid defining variables like `TARGET_LIBRARY` that simply expand to another variable. It's better to use that variable directly.

While I appreciate the intent to avoid repetition by setting the library name to the project name, I think this aspect would be better handled by a linter than introducing this complexity.